domain-models-runtime-it/src/main/java/org/folio/rest/impl/MyitemsApi.java

```diff
@@ -49,4 +49,11 @@ public void putMyitemsByMyitemId(String id, String lang, Myitem entity, Map<Stri
     PgUtil.put(TABLE, entity, id, okapiHeaders, vertxContext,
         PutMyitemsByMyitemIdResponse.class, asyncResultHandler);
   }
+
+  @Override
+  public void patchMyitemsByMyitemId(String id, Myitem entity, Map<String, String> okapiHeaders,
+                                     Handler<AsyncResult<Response>> asyncResultHandler, Context vertxContext) {
+    PgUtil.put(TABLE, entity, id, okapiHeaders, vertxContext,
```

Comment:
As far as I understand PgUtil.put method will completely overwrite an existing record. Probably it is needed to implement new PgUtil.patch(...) method. Please, correct me if I wrong.

Reply:
This issue and PR is about RAML and RestVerticle only.
This is needed for https://issues.folio.org/browse/MODKBEKBJ-447 .
Adding PATCH support in PostgresClient or PgUtil is out of scope.

Reply:
The main goal of this PR is to have RMB ability to process PATCH requests. 

`MyItemApi` is using only for test purposes. 
I think that implementing PgUtil.patch(...) is out of scope. If you think that this method is needed than I can create a Jira ticket for it.

Reply:
It would be nice to add a comment in the code about using PgUtil.put to cover patch method for the time being.

Reply:
@hjiebsco 
Comment was added